src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -480,7 +476,9 @@
                 (swap! state assoc
                        ;; vector only used when checking redaction, N/A for Dockstore
                        :methods {[nil nil] (mapv :name (get-parsed-response))})
-                (swap! state assoc :methods {} :error-message status-text))))))))
+                (swap! state assoc :methods {} :error-message status-text))))))
+       (if-not (:entities-loaded? @locals)
+         (this :-load-entities))))
```

Comment:
So this was happening because `:-load-entities` was not being called if it was fetching for dockstore (but worked fine for agora)?

Reply:
yes. and for the dockstore case, when it tried to go into edit mode (i.e. someone clicking on edit, or the entity model checkbox or populate with json), it would show a spinner saying loading... and then return without actually going into edit mode.